python-sdk/tests_integration/databases/test_all_databases.py

```diff
@@ -153,50 +190,9 @@ def test_load_pandas_dataframe_to_table_with_replace(database_table_fixture):
         target_table=table,
     )
 
-    statement = f"SELECT * FROM {database.get_table_qualified_name(table)};"
-    response = database.run_sql(statement)
-    rows = response.fetchall()
+    rows = database.fetch_all_rows(table)
     assert len(rows) == 2
     assert rows[0] == (3,)
     assert rows[1] == (4,)
 
     database.drop_table(table)
-
-
-@pytest.mark.integration
-@pytest.mark.parametrize(
-    "database_table_fixture",
-    [
-        {
-            "database": Database.BIGQUERY,
-            "table": Table(metadata=Metadata(schema=SCHEMA)),
-        },
-        {
-            "database": Database.POSTGRES,
-            "table": Table(metadata=Metadata(schema=SCHEMA.lower())),
-        },
-        {
-            "database": Database.SNOWFLAKE,
-            "table": Table(metadata=Metadata(schema=SCHEMA)),
-        },
-        {"database": Database.SQLITE, "table": Table()},
-    ],
-    indirect=True,
-    ids=["bigquery", "postgres", "snowflake", "sqlite"],
-)
-@mock.patch("astro.files.base.File.export_to_dataframe")
-@mock.patch("astro.files.base.File.export_to_dataframe_via_byte_stream")
-def test_export_to_dataframe_via_byte_stream_is_called_for_postgres(
```

Comment:
Why did we delete this?

Reply:
It was just moved, @kaxil !

Reply:
aah although why :D ?

Reply:
@kaxil this probably happened during a merge from main. I'll move it back tomorrow morning